### PR TITLE
Eventing 0.14 in 1.8 CSV

### PR DESCRIPTION
**Pull request by aliok**
Changes:
- 0.14 eventing images in 1.8 CSV
- Adaptions for that (deployment names to check is changed; namespace deletion in tooling is changed to proceed in case there's a completed Kubernetes Job pod)
- Install Zipkin and use it in Eventing (needed to pass the tracing related e2e and conformance tests)

### CODE REVIEW

**Comment by aliok** [comment]
Getting this:

```
imc-controller-84f97dcb7b-bt4jk controller {"level":"fatal","ts":"2020-05-13T08:13:16.845Z","logger":"controller","caller":"sharedmain/main.go:174","msg":"Failed to start configuration manager","error":"configmap \"config-domain\" not found","stacktrace":"knative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain.MainWithConfig.func2\n\t/go/src/knative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain/main.go:174\nknative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain.MainWithConfig\n\t/go/src/knative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain/main.go:195\nknative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain.MainWithContext\n\t/go/src/knative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain/main.go:128\nknative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain.Main\n\t/go/src/knative.dev/serving/vendor/knative.dev/pkg/injection/sharedmain/main.go:122\nmain.main\n\t/go/src/knative.dev/serving/cmd/controller/main.go:34\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"}
```

**Comment by aliok** [comment]
```
=== RUN   TestActivatorHA
--- FAIL: TestActivatorHA (60.25s)
    activator_test.go:50: Deployment activator not scaled to 2: timed out waiting for the condition
```

File: hack/lib/tracing.bash [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+function install_tracing {
+  deploy_zipkin
+  enable_eventing_tracing
+}
+
+function deploy_zipkin {
```
Comment (markusthoemmes) [inline hack/lib/tracing.bash:8]: Shouldn't we use Openshift Tracing (Jaeger) here?

Comment (aliok) [reply hack/lib/tracing.bash:8]: No idea. Just trying to make things work honestly.
Maybe we can do that later

**Comment by aliok** [comment]
/retest

Throttling issues at AWS

File: hack/lib/tracing.bash [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      namespace: ${ZIPKIN_NAMESPACE}
+    spec:
+      type: NodePort
+      ports:
+      - name: http
+        port: 9411
+      selector:
+        app: zipkin
+    ---
+    apiVersion: apps/v1
```
Comment (markusthoemmes) [inline hack/lib/tracing.bash:24]: Fix the indentation here? I believe it can all start at the left-most character.

File: hack/lib/tracing.bash (hunk 1 above)
Comment (aliok) [reply hack/lib/tracing.bash:8]: I will create a JIRA for it after this PR is merged

File: hack/lib/namespaces.bash [hunk 3]
```diff
@@ -15,7 +15,7 @@ function delete_namespaces {
   for ns in "${NAMESPACES[@]}"; do
     if oc get ns "${ns}" >/dev/null 2>&1; then
       logger.info "Waiting until there are no pods in ${ns} to safely remove it..."
-      timeout 600 "[[ \$(oc get pods -n $ns -o jsonpath='{.items}') != '[]' ]]"
+      timeout 600 "[[ \$(oc get pods -n $ns --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
```
Comment (markusthoemmes) [inline hack/lib/namespaces.bash:18]: Why are these changes needed?

Comment (markusthoemmes) [reply hack/lib/namespaces.bash:18]: Ah, I shall read the description better :D.

**Comment by markusthoemmes** [comment]
```
./test/e2e-tests.sh: line 40: install_tracing: command not found
```

**Comment by aliok** [comment]
https://github.com/openshift-knative/serverless-operator/pull/301 is relevant.

We can merge this one and then change the "replaces" to 1.7.1 once 1.7.1 CSV is in master

**Comment by matzew** [comment]
/close 

in fav. of #328
